docker/base.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+set -xe
+
+# Make sure debconf is in noninteractive mode
+export DEBIAN_FRONTEND=noninteractive
+
+# Hold grub-pc because it can't be upgraded in headless mode
+echo grub-pc hold | dpkg --set-selections
+
+# Make sure our packages list is updated
+apt-get update
+
+# We install an up to date git version because we might be using commands only
+# present in modern versions. For this we need python-software-properties /
+# software-properties-common packages that provide add-apt-repository (for
+# easily installing PPAs).
+apt-get -y install apt-transport-https build-essential bzip2 devscripts \
+    sudo debhelper less lsb-release vim wget curl adduser \
+    python3 python-docutils python-software-properties \
+    software-properties-common
+
+# Get the Ubuntu codename
+release=`lsb_release -cs`
+
+# Install packages that depends on the distro
+ruby=ruby1.9.1-dev
+test $release = xenial &&
```

Comment:
Not now, but the hardcoded trusty build is just a simplification. In a near future I would want to build for both trusty and xenial (also, this distinction here is not even necessary, there is a `ruby-dev` package that works for both :grin: )